components/brave_wallet_ui/components/desktop/popup-modals/edit-visible-assets-modal/index.tsx

```diff
@@ -385,6 +390,21 @@ const EditVisibleAssetsModal = (props: Props) => {
     setShowAdvancedFields(!showAdvancedFields)
   }
 
+  const onShowNetworkDropDown = () => {
+    setShowNetworkDropDown(true)
+  }
+
+  const onHideNetworkDropDown = () => {
+    if (showNetworkDropDown) {
```

Comment:
`showNetworkDropDown` is a dependancy. Use a `useCallback`, or maybe just remove the `if`

Reply:
Updated!